app-emulation/retroarch/retroarch-1.7.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic
+
+DESCRIPTION="A sophisticated libretro frontend for emulators, game engines and media players."
+HOMEPAGE="http://retroarch.com/"
+SRC_URI="https://github.com/libretro/RetroArch/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~arm"
+IUSE="+opengl +pulseaudio -alsa -sdl -X cg"
+
+DEPEND="
+	dev-libs/libxml2
+	>=media-libs/freetype-2.8
+	opengl? ( virtual/opengl )
+	X? ( x11-libs/libX11 x11-apps/xinput )
+	pulseaudio? ( media-sound/pulseaudio )
+	alsa? ( media-sound/alsa-utils )
+	sdl? ( media-libs/libsdl2 )"
+
+RDEPEND="
+	${DEPEND}
+	x86? ( cg? ( media-gfx/nvidia-cg-toolkit ) )
```

Comment:
why arch dependent?

Reply:
The nvidia-cg-toolkit package is only available for x86/amd64 systems. It is not essential for RetroArch, but can be used for extra features if it exists.

https://packages.gentoo.org/packages/media-gfx/nvidia-cg-toolkit